vertx-codegen-processor/src/main/java/io/vertx/codegen/processor/Processor.java

```diff
@@ -155,6 +152,15 @@ private Path determineSourcePathInEclipse() {
     return null;
   }
 
+  private String throwableToMessageString(Throwable e) {
+    StringBuilder builder = new StringBuilder();
```

Comment:
instead use `e.printStackTrace(new PrintWriter(new StringWriter()))`

Reply:
Let me do this